robots/barr-e/python/barr_e/workloads/mind/mind_test_workload.py

```
@@ -0,0 +1,272 @@
+        self.session_id = _safe_str(self.rsc.get("id", ts.get("topic", "session")))
+        self.sequence = [_as_dict(s) for s in _as_list(ts.get("sequence", []))]
+
+        _log(f"Loaded {len(self.sequence)} steps from {rsc_path}")
+
+        if self.sequence:
+            self._set_target_from_step(self.sequence[0])
+            for k in self.current_state:
+                if abs(self.current_state[k] - self.target_state[k]) < 1e-9:
+                    self.current_state[k] -= 1e-6
+
+    def _restart(self, reset_state=False):
```

Ruff (0.15.2)

[warning] Missing return type annotation for private function `_restart`

Add return type annotation: `None`

(ANN202)